lib/rules/disallow-space-after-object-keys.js

```diff
@@ -134,7 +134,7 @@ module.exports.prototype = {
             var maxKeyEndPos = 0;
             var tokens = [];
             node.properties.forEach(function(property) {
-                if (property.shorthand || property.kind !== 'init') {
+                if (property.shorthand || property.kind !== 'init' || property._babelType === 'SpreadProperty') {
```

Comment:
This is done because in babel-jscs we change the SpreadProperty node type https://github.com/jscs-dev/babel-jscs/blob/1d15f1afa7dc7af24726187fcfb58cf9cdc4ff38/acorn-to-esprima.js#L181-L189